app/controllers/user_panel/orders_controller.rb

```diff
@@ -0,0 +1,37 @@
+module UserPanel
+  class OrdersController < ApplicationController
+    before_action :authenticate_user!
+
+    def index
+      @orders = current_user.orders
+    end
+
+    def show
+      @order = attach_order
+    end
+
+    def confirm
+      order = current_user.orders.pending.find(params[:id])
+      order.update(
```

Comment:
skoro OrderItem ma kolumnę z ceną:
```ruby
class OrderItem < ApplicationRecord
  monetize :price_cents
  belongs_to :product
end
```

to w momencie składania zamówienia ta cena powinna zostać przypisana